modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/util/ObjectEntryVariablesUtil.java

```diff
@@ -226,7 +286,8 @@ private static Map<String, Object> _getVariables(
 		DTOConverterRegistry dtoConverterRegistry,
 		ObjectDefinition objectDefinition, JSONObject payloadJSONObject,
 		SystemObjectDefinitionMetadataRegistry
-			systemObjectDefinitionMetadataRegistry) {
+			systemObjectDefinitionMetadataRegistry,
+		boolean original) {
```

Comment:
@p-albuquerque please sort this alphabetically.. In addition, maybe we can rename this to `includeOriginalObjectEntry` or something more intuitive